checkov/terraform/parser.py

```diff
@@ -577,6 +619,21 @@ def _clean_parser_types(conf: dict[str, Any]) -> dict[str, Any]:
                 sorted_conf[attribute] = str(values)
         return sorted_conf
 
+    def get_file_key(self, file, nested_data):
+        if not nested_data:
+            return f'{file}'
+        nested_str = self.get_file_key(nested_data.get("file"), nested_data.get('nested_modules_data'))
+        nested = f'{file}[{nested_str}#{nested_data.get("module_index")}]'
```

Comment:
@achiar99 Can we refactor the path creation logic into a function? (`{key}[file]#{module_index}`)